pyproject.toml

```diff
@@ -10,8 +10,6 @@ authors = [
 ]
 keywords = ["pytorch", "finetuning", "llm"]
 dependencies = [
-    # multimodality
-    "torchvision",
```

Comment:
Just chatted with @NicolasHug on this and he confirmed it's not possible since there's no way to point pyproject.toml to a specific conda channel or PyPI repo

Reply:
got it! i am just afraid that that in the long run it may cause issues, since we may want to pin torchvision version. For example, in ClipTransforms, older versions will break. 

brainstorming:
Maybe we can also force it to also install torchvision nightlies, like you force pytorch nightlies?

For example, in the [link you shared](https://gist.github.com/ebsmothers/dc78853f03d89c156673a9e84a321238), it installed version 0.18. [But nightly is 0.20](https://pytorch.org/vision/versions.html).

Reply:
It should be installed in a single command, this is why I updated the readme to clarify this. So if the user runs `pip install torch torchvision` they will get stable versions of both torch and torchvision; if they run `pip3 install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cu121` they will get torch nightly and torchvision 0.20. 

I can't find the link to the original comment but I believe that was demonstrating what happens when we install using the existing pyproject.toml. For reference, [here](https://gist.github.com/ebsmothers/5f377798d834fc76b4823c5557584c08) is my pip list after running the first command; [here](https://gist.github.com/ebsmothers/9d4ed23ab72955be639fdc20d03a5809) is my pip list after running the second command. You can see that the versions are as expected.

Btw regarding pinning versions -- we do not test on anything older than the latest stable version of PyTorch and I don't think we want to worry about breaking folks on older versions than that. By the same logic, I don't think we should be pinning to older versions of torchvision. Then the best way to keep things in sync is just install the two together using these commands.

Reply:
thanks for the links and explanation!